File: app/src/main/java/com/example/tugasdua/MainActivity.java
package com.example.tugasdua;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import com.google.android.material.bottomnavigation.BottomNavigationView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        BottomNavigationView bottomNavigationView = findViewById(R.id.bottom_navigation);

        bottomNavigationView.setOnNavigationItemSelectedListener(item -> {
            switch (item.getItemId()) {
                case R.id.nav_home:
                    break;
                case R.id.nav_profile:
                    startActivity(new Intent(MainActivity.this, ProfileActivity.class));
                    break;
                case R.id.nav_repository:
                    break;
            }
            return true;
        });
    }
}

File: app/src/main/java/com/example/tugasdua/MainActivity.kt
package com.example.tugasdua

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

// Sesuaikan dengan package name proyek kamu
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // Menampilkan teks nama dan kelas
        val textView = findViewById<TextView>(R.id.textViewNama)
        textView.text = "Andre Johann Jonnius - 4B."
    }
}